README.md

```diff
@@ -110,44 +110,40 @@ With the `updateId`, you can check the status (`enqueued`, `processed` or `faile
 
 ```php
 // MeiliSearch is typo-tolerant:
-print_r($index->search('harry pottre'));
+print_r($index->search('harry pottre')->getHits());
```

Comment:
What would the complete instance look like? Without getHits ?

Reply:
It's an instance, it does not look anything readable, it's too long.
And in some languages, an instance is not even displayable without using a method in the class.